source/la.html.erb

```diff
@@ -1,285 +1,339 @@
-            </p>
+    <section id="sponsors" class="sponsor-section">
+      <h2>Sponsors</h2>
+
+      <section class="sponsor-list">
+        <div class="row">
+          <div class="sponsors crevalle">
+            EMPEX is a production of <br />
+            <a class="image-link" target="_blank" href="http://crevalle.io/">
+              <%= image_tag "sponsors/crevalle.png", alt: "Crevalle", class: "crevalle-logo" %>
+            </a>
           </div>
         </div>
-      </section>
 
-    </section>
-  </section>
+        <%# <div class="row">
+          <div class="sponsors platinum">
+            <h3>Platinum</h3>
 
-  <section id="schedule" class="schedule">
-    <h2>Schedule</h2>
-    <table class="schedule-table">
-      <tr>
-        <td class="event">
-          <div class="title">Doors open / Registration</div>
-          Show up and hang out.  Mingle!
-        </td>
-        <td class="time">
-          9:00AM
-        </td>
-      </tr>
-      <tr>
-        <td class="event">
-          <div class="title">Morning presentations</div>
-          Awesome people presenting.
-        </td>
-        <td class="time">
-          9:30AM
-        </td>
-      </tr>
-      <tr>
-        <td class="event">
-          <div class="title">
-            Lunch
           </div>
-          Yum yums.
-        </td>
-        <td class="time">
-          12:00PM
-        </td>
-      </tr>
-      <tr>
-        <td class="event">
-          <div class="title">
-            Afternoon presentations
+        </div> %>
+
+        <div class="row">
+          <div class="sponsors gold">
+            <h3>Gold</h3>
+
+            <a class="image-link" target="_blank" href="http://carbonfive.com">
+
+              <%= image_tag "sponsors/carbonfive.svg", alt: "Carbon Five" %>
+            </a>
           </div>
-          More quality content.
-        </td>
-        <td class="time">
-          1:00PM
-        </td>
-      </tr>
-      <tr>
-        <td class="event">
-          <div class="title">Closing remarks</div>
-          Let's wrap it up!
-        </td>
-        <td class="time">
-          5:00PM
-        </td>
-      </tr>
-      <tr>
-        <td class="event">
-          <div class="title">
-            After party
+        </div>
+
+        <%# <div class="row">
+          <div class="sponsors silver">
+            <h3>Silver</h3>
+
           </div>
-          Some drinks to wash down the learning.
-        </td>
-        <td class="time">
-          6:00PM
-        </td>
-      </tr>
-    </table>
-  </section>
+        </div> %>
+      </div>
+    </section>
 
-  <div class="hr hr--white"></div>
+    <div class="footer">
+      <div>
+        If you're interested in sponsoring EMPEX, please see the
+        <a href="https://docs.google.com/document/d/1meg5KshORij9x6nwda8Oh4zmEl70sWpN7reqcKS5tYs/edit">prospectus</a>,
+        or email
+        <a mailto: 'info@empex.co'>info@empex.co</a>.
```

Comment:
does this syntax work? it's not a function call.

Reply:
oh yeah weird, I don't know what happened there, copied without fixing I guess.